docs/config-definition.md

```diff
@@ -50,7 +45,7 @@ The configurable parameters are as follows:
 
 | Key | Type | Default | Meaning |
 | --- | --- | --- | --- |
-| `persistentVolumeProvider` | CloudProviderConfig<br><br>(Supported key values are `aws`, `gcp`, and `azure`, but only one can be present. See the corresponding [AWS][0], [GCP][1], and [Azure][2]-specific configs.) | Required Field | The specification for whichever cloud provider the cluster is using for persistent volumes (to be snapshotted).<br><br> *NOTE*: For Azure, your Kubernetes cluster needs to be version 1.7.2+ in order to support PV snapshotting of its managed disks. |
+| `persistentVolumeProvider` | CloudProviderConfig<br><br>(Supported key values are `aws`, `gcp`, and `azure`, but only one can be present. See the corresponding [AWS][0], [GCP][1], and [Azure][2]-specific configs.) | None (Optional) | The specification for whichever cloud provider the cluster is using for persistent volumes (to be snapshotted), if any.<br><br>If not specified, Backups and Restores requesting PV snapshots & restores, respectively, are considered invalid. <br><br> *NOTE*: For Azure, your Kubernetes cluster needs to be version 1.7.2+ in order to support PV snapshotting of its managed disks. |
```

Comment:
Have we tested this in conjunction with the other PR https://github.com/heptio/ark/pull/45 that sets `--snapshot-volumes` and `--restore-volumes` to default true?

The author of that PR said it was fine to leave the quickstart commands without `--snapshot-volumes=false` since the basic Nginx example doesn't have PVs. I'm wondering if we actually do need to specify this flag now in the sample commands, since it looks it will error without a `peristentVolumeProvider` section specified. It's not a big deal, but if that's the case, we'll need to make sure to change the backup/restore commands in one of these two PRs. @ncdc does that seem right?